pkg/cluster/cluster.go

```diff
@@ -377,13 +377,26 @@ func (c *Cluster) createPodAndService(members etcdutil.MemberSet, m *etcdutil.Me
 		token = uuid.New()
 	}
 	pod := k8sutil.MakeEtcdPod(m, members.PeerURLPairs(), c.Name, state, token, c.Spec)
+	pod = k8sutil.PodWithOwnerRef(pod, c.makeOwnerRef())
 	if needRecovery {
 		k8sutil.AddRecoveryToPod(pod, c.Name, m.Name, token, c.Spec)
 	}
 	_, err := c.KubeCli.Pods(c.Namespace).Create(pod)
 	return err
 }
 
+func (c *Cluster) makeOwnerRef() k8sapi.OwnerReference {
```

Comment:
can we make this a util func in pkg cluster? i do not feel good about adding tons of methods directly on cluster.

createOwnerRef(c *Cluster) k8sapi.OwnerReferene?